libs/docs/src/docs/no-version/helloworld-example-odo.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Hello world example: odo v3"
+description: "Hello world example: odo v3"
+---
+
+This guide will run through creating a simple hello world devfile project using `odo` version 3. The purpose of this guide is to provide a practical introduction to those just starting out with devfiles.
+
+## Perquisites
+
+- `odo` v3
+- `kubectl` or `oc`
+- Docker 17.05 or higher
+- Access to a Kubernetes or OpenShift cluster
```

Comment:
We do call out that a cluster is required, but I accidentally glossed over this point when I went through the guide. Maybe we can add an optional step to start a minikube cluster?

Reply:
> > We do call out that a cluster is required, but I accidentally glossed over this point when I went through the guide. Maybe we can add an optional step to start a minikube cluster?
> 
> I can add this step for the `odo` guide.

Actually I can add this in both.